includes/sdk-troubleshooting-and-debugging/browser.md

```diff
@@ -61,8 +61,14 @@ Cross-origin resource sharing (CORS) prevents a malicious site from reading anot
 
 If you have set up an API proxy and run into configuration issues related to that on a platform you’ve selected, that’s no longer an SDK issue but an integration issue between your application and the service provider.
 
-##### Event Dropping When Closes the Browser or Leaves the Page
+##### Events fired but no network requests
+
+If you [set the logger to "Debug" level](./#debug-mode), and see track calls in the developer console, the `track()` method has been called. If you don't see the corresponding event in Amplitude, the Amplitude Instrumentation Explorer Chrome extension, or the network request tab of the browser, the event wasn't sent to Amplitude. Events are fired and placed in the SDK's internal queue upon a successful `track()` call, but sometimes these queued events may not send successfully. This can happen when an in-progress HTTP request is cancelled. For example, if you close the browser or leave the page with a `history.push()` event in React.
+
+There are two ways to address this issue:
+
+1. If you use standard network requests, set the transport to `beacon` during initialization or set the transport to `beacon` upon page exit. `sendBeacon` doesn't work in this case because it sends events in the background, and doesn't return server responses like `4xx` or `5xx`. As a result, it doesn't retry on failure. `sendBeacon` sends only scheduled requests in the background. For more information, see the [sendBeacon](./#use-sendbeacon) section.
+
+2. To make track() synchronous, [add the `await` keyword](./#callback) before the call.
```

Comment:
can we also let them print out the return result of those promises to make sure those calls return 200?

Reply:
@yuhao900914 Does this block merging of this PR?

cc: @Mercy811